### ISSUE TITLE
Interp Viewer, timeline jumpy

### ISSUE THREAD

**Issue by ghost** [ME3Explorer/ME3Explorer#337]
@SirCxyrtyx -- I know you've basically stopped development on this tool (well, I think), but I just wanted to let you know that the timeline still really likes to jump around. It seems to be linked to scrolling and window adjusting.

To replicate, try this:
- Load a PCC that has an interpdata with lots of interpgroups. A good one is BIoD_ProMar_750Finale.pcc
- Scroll down a bit and select a group to expand it, like, HMF_FemShep_.
- Now click on a track, like the one for the sub.
- Now scroll down again just a little, and then widen the whole window.

Doing that breaks the timeline and makes it jump near the top of the display area. It will stay there as I continue to scroll. Adjusting the size of the window _again_ makes it disappear completely, and scrolling doesn't make it re-appear. If I reload the interpdata, then it _will_ reappear.

**Comment by SirCxyrtyx** [comment]
I wrote the InterpViewer when I was a somewhat less experienced programmer, and it shows. It's very poorly coded, and quite frankly, it's astonishing that it works as well as it does. I just recently tried to add support for showing SFXSceneshop groups, and it broke everything in ways that completely baffled me. The first post-stable release will show the groups, but in much more limited fashion than I wanted.

At this point I really just want to burn the tool to the ground and rewrite it. Once I learn WPF, I'll do just that. So for now, I'd advise just not resizing the window after opening an interpdata.
